src/ui/GeoHash.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Provide functionality for GeoHash
+ * @constructor
+ */
+WGL.ui.GeoHash = function () {
+  const base32 = '0123456789bcdefghjkmnpqrstuvwxyz';
+  const lon_min = -180.0;
+  const lon_max = 180.0;
+  const lat_min = -90.0;
+  const lat_max = 90.0;
+
+  /**
+   * Encode position to the GeoHash
+   * @param lon {number} longitude
+   * @param lat {number} latitude
+   * @param level {int} number of char in GeoHash
+   * @returns {string} geohash
+   */
+  this.encode = function (lon, lat, level) {
+    let a_lon = lon_min;
+    let b_lon = lon_max;
+
+    let a_lat = lat_min;
+    let b_lat = lat_max;
+
```

Comment:
Is there any reason why not to use lon_min directly? Using the lon_min will do it more readable I guess.

Reply:
it is due to general